test/tests/error-handling-prod.test.js

```diff
@@ -47,20 +47,18 @@ describe('graphql - error handling in production', () => {
       const log = console.warn.mock.calls[0][1] || JSON.parse(console.warn.mock.calls[0][0])
 
       if (cds.env.features.cds_assert) {
-        // REVISIT: message missing, cds.assert also misses some properties like type, element, entity ...
```

Comment:
those properties are not needed. as long as there is a target, the developer can look up the rest.